components/dashboard/src/components/Menu.tsx

```diff
@@ -57,10 +60,27 @@ function Menu(props: { left: Entry[], right: Entry[] }) {
                         {props.right.map(MenuItem)}
                     </ul>
                 </nav>
-                <Link className="lg:ml-3 flex items-center justify-start lg:mb-0 mb-4 pointer-cursor m-l-auto rounded-full border-2 border-white hover:border-gray-200 p-0.5" to="/account">
-                    <img className="rounded-full w-6 h-6"
-                        src={user?.avatarUrl || ''} alt={user?.name || 'Anonymous'} />
-                </Link>
+                <div className="lg:ml-3 flex items-center justify-start lg:mb-0 mb-4 pointer-cursor m-l-auto rounded-full border-2 border-white hover:border-gray-200 p-0.5">
```

Comment:
**question**: Could we make the border gray-500 when the profile menu is active (dropdown is visible)?

Reply:
Seems rather involving to do tbh

Reply:
True, let's drop the idea for now.